ui/src/main/java/org/pentaho/mantle/client/dialogs/scheduling/OutputLocationUtils.java

```diff
@@ -44,7 +45,7 @@ public static void validateOutputLocation( final String outputLocation, final Co
 
     final String url = EnvironmentHelper.getFullyQualifiedURL()
       + "plugin/scheduler-plugin/api/generic-files/folders/"
-      + GenericFileNameUtils.encodePath( outputLocation );
+      + URL.encodeQueryString( GenericFileNameUtils.encodePath( outputLocation ) );
```

Comment:
I think it would be preferable to use our `NameUtils#URLEncode` method: https://github.com/pentaho/pentaho-commons-gwt-modules/blob/9fb6c022eef51c0c03164b5963ec6dd91aa409e1/widgets/src/main/java/org/pentaho/gwt/widgets/client/utils/NameUtils.java#L79.